packages/stargate/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "@cosmjs/encoding": "^0.22.0",
+    "@cosmjs/launchpad": "^0.22.0",
```

Comment:
???

Reply:
We use `import { Coin } from "@cosmjs/launchpad";`

Reply:
Why don't we have `Coin` in Stargate?

Reply:
We can also have a copy of the type, no problem. In the meantime we use a few more amino helpers from `@cosmjs/launchpad` in `@cosmjs/stargate` which are temporary (hopefully). But I agree, it would be nice to remove this dependency at some point, either by copying what we need or by extrating commin symbols in a shared package.